lib/screens/v2/widget_instance/train_crowding.ex

```diff
@@ -0,0 +1,90 @@
+defmodule Screens.V2.WidgetInstance.TrainCrowding do
+  @moduledoc """
+  A widget that displays the crowding on a train that is en route to the current station.
+  """
+
+  alias Screens.Config.Screen
+  alias Screens.Config.V2.Triptych
+  alias Screens.Predictions.Prediction
+
+  defstruct screen: nil,
+            prediction: nil,
+            now: nil
+
+  @type t :: %__MODULE__{
+          screen: Screen.t(),
+          prediction: Prediction.t(),
+          now: DateTime.t()
+        }
+
+  @type widget_data :: %{
+          destination: String.t(),
+          crowding: list(crowding_level),
+          platform_position: number,
+          front_car_direction: :left | :right,
+          now: String.t()
+        }
+
+  @type crowding_level :: :no_data | :not_crowded | :some_crowding | :crowded | :disabled
+
+  @spec serialize(t()) :: widget_data()
+  def serialize(%__MODULE__{
+        screen: %Screen{app_params: %Triptych{train_crowding: train_crowding}},
+        prediction: prediction,
+        now: now
+      }) do
+    %{
+      destination: prediction.trip.headsign,
+      crowding: serialize_carriages(prediction.vehicle.carriages),
+      platform_position: train_crowding.platform_position,
+      front_car_direction: train_crowding.front_car_direction,
+      now: serialize_time(now)
+    }
+  end
+
+  defp serialize_time(%DateTime{} = time) do
+    DateTime.to_iso8601(time)
```

Comment:
Does this need to be timezone shifted? Or were you thinking that can be done on the client?

Reply:
I mimicked the serializer for AlertHeader. And it seems to be translating correctly with the <Time> component on the frontend, so I think we're good